annotate.py

```diff
@@ -212,20 +214,19 @@ def annotate_img(img_path, initials, size1=1, size2=1) :
 
     #Make mask same colour as drawing and output binarised image
     train_labels = img[:,:,:3]
-    lower_blue = np.array([254,0,0], dtype = "uint16")
-    upper_blue = np.array([255,0,0], dtype = "uint16")
-    mask_blue = cv2.inRange(train_labels, lower_blue, upper_blue)
-    mask_blue[mask_blue < 250] = 0
-    #mask_blue[mask_blue != 0 ] = 255
-    mask_blue[mask_blue != 0 ] = 1
+    lower_red = np.array(RED_MIN, dtype = "uint16")
```

Comment:
The code seems to work if both the upper and lower limits are equal (both `255` instead of a range).

We could just have:
`   upper_red = np.array(RED_MAX, dtype = "uint16")
    mask_red = cv2.inRange(train_labels, upper_red, upper_red)`

Do we still need to find `RED_MIN`?

Reply:
nope, even better.

Reply:
Ok nice that simplifies things.
(I'm using this method in `find_centroids.py` too)